app/ui/src/app/integration/integration_detail/integration-activity/integration-activity.component.ts

```diff
@@ -97,6 +99,17 @@ export class IntegrationActivityComponent implements OnInit {
     this.showPagination = (this.allActivities.length > this.paginationConfig.pageSize);
     this.paginationConfig.totalItems = this.allActivities.length;
     this.paginationConfig.pageNumber = 1;
+
+    this.openshiftConsoleURL = null;
+    if (this.allActivities.length > 0) {
+      const base = this.configService.getSettings('consoleUrl');
```

Comment:
@deeleman @gashcrumb that's where this setting is queried (it's configured as an empty string, non-null).

Reply:
Guess the configService is used wrongly here anyway (sorry, took this over blindly from the original PR). 
Going to fix this.